README.md

```diff
@@ -1,119 +1,98 @@
+- **Ibis**: Provides a unified interface for data manipulation
+- **DuckDB**: In-memory analytical database for fast data processing
+- **Parquet**: Columnar storage format for efficient data storage
+- **S3**: Cloud storage for data lake architecture
 
 ## How It Works
+
+### Pipeline Process
 The data pipeline consists of three main stages:
 
-### Ingestion Process (`ingest/run.py`)
+#### Ingestion Process (`ingest/run.py`)
 - Reads raw CSV files from `datalake/raw/<source>` directories
 - Converts them to Parquet format using DuckDB
 - Uploads the Parquet files to S3 under `<env>/landing/<source>/` folders
-- Creates separate folders for different data sources (edu, wdi)
+- Creates folders for each data source:
+  - `edu`: Education-related datasets
+  - `wdi`: World Development Indicators
 
-### Transformation Process (SQLMesh)
+#### Transformation Process (SQLMesh)
 - Reads Parquet files from the S3 landing zone
-- Performs transformations using SQLMesh models:
+- Performs data transformations using SQLMesh models
 - Stores transformed data in local DuckDB database (`osaa_mvp.db`)
 - Outputs transformed data to S3 under `<env>/transformed/<schema>/` folders
 
-### Upload Process (`upload/run.py`)
+#### Upload Process (`upload/run.py`)
 - Takes transformed data from the DuckDB database
 - Uploads final transformed datasets to S3 under `<env>/transformed/` directory
-- Currently focuses only on uploading WDI (World Development Indicators) transformed data
+- Currently focuses on uploading WDI (World Development Indicators) transformed data
+
+### Environment Configuration
+The pipeline supports different execution environments controlled through environment variables:
+- `TARGET`: Controls S3 paths and SQLMesh environments (`dev`, `int`, `prod`). Default is `dev`
+- `USERNAME`: Used for S3 paths in `dev` environment. Default is `default`
 
-The environment (`<env>`) in S3 paths is determined by configuration:
-   - Production: `prod/`
-   - Integration: `int/`
-   - Development: `dev_<username>/`
+S3 path examples:
+- Production: `prod/`
+- Integration: `int/`
+- Development: `dev_<username>/`
 
 ## Getting Started
 
-### Prerequisites
-- Python versions 3.9 to 3.11 (3.12 not supported)
+### User Profiles
+1. **Data Analyst/Policy Analyst** - You are a user of the data, and you need to run the data pipeline in your own development environment. Link to Instructions
+2. **Data Engineer** - You are a project maintainer and you need to make edits to the pipeline process. You will interact directly with the code and cli tools, in addition to the compiling and running the container. Link to Instructions.
+
+
+### Process for Data Analysts
+Please follow this section if your role is Data Analysts. Please see the definition of [Data Analyst](#user-profiles) above for more details.
+
+#### Prerequisites
+- AWS account with S3 access
+- Docker Desktop - for building and running the pipeline in a containerized environment
+- Git cli
```

Comment:
When I say data analysts in this context I really mean Policy Analysts, which are the users who are currently using excel to process manual downloads. I'd like to give them a straightforward process to follow, so the less requirements the better.

I figured the data engineers would just skip to their own section and so it would be good to have the full set there too

Reply:
I believe the simplest option is to have them use docker and run the commands that way. I didn't include the just cli here for that reason

Reply:
Okay, that makes sense!